common/types/cards.ts

```diff
@@ -60,3 +63,20 @@ export type SlotPos = {
 	row: RowState
 	slot: BoardSlot
 }
+
+export type CardLogFactory = {
+	player: string
+	coinFlip: string
+	header: string
+	pos: {
+		rowIndex: string
+		name: string
+		hermitCard: string
+		slotType: string
+	}
+	pick: {
+		rowIndex: string
+		name: string
+		slotType: string
+	}
+}
```

Comment:
Sorry this was my bad. This type name is awful. I am not sure what a good type name is.
Also please document the fields.